### PR TITLE
add 3rd-Party installation method

### PR DESCRIPTION
**Pull request by codenoid**
add 3rd-Party installation method via gobinaries.com

### CODE REVIEW

**Comment by aligator** [comment]
Thanks, didn't know that service, yet :-)

But I have some things to discuss:
1. Is this really needed as we provide releases with built binaries (for the most important architectures and systems) very regularly? (If any specific architecture+os combination is needed, just create a issue.)
2. One problem with this could be that this service does not inject the version, which is a problem if someone creates a issue and tries to use `verless -v`

To solve 2. we could just add a info that these builds are not supported and issues should be always tested using official releases.

**Review by aligator — CHANGES_REQUESTED** [review]
To be discussed

**Review by dominikbraun — CHANGES_REQUESTED** [review]
> To solve 2. we could just add a info that these builds are not supported and issues should be always tested using the latest official release.

Let's do that and we're good to go.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,14 @@ another command, just append it to the image name:
 $ docker container run verless/verless version
 ```
 
+### 3rd-Party method
+
```
Comment (aligator) [inline README.md:80]:
```suggestion

__Unsupported, use the official builds when reporting issues!__
```

**Pull request closed without merging**